app/models/question.rb

```diff
@@ -151,18 +152,29 @@ def build_complete_requirements
   end
 
   def full_number
```

Comment:
Is this still named `full_number` to avoid renaming in code using this, or some other reason? It seems like what this function returns is no longer actually 'full_number.' The full number would look like III.2.A, for example. If we want this function to do what it's doing here, could we rename it to something like outline_head (I looked this up in wikipedia  . .. )? I can imagine a request from tww to do a 'full_number' on leaf nodes that would actually have III.2.A.ii etc. I think  that updating the name would help us adjust as we get feedback and with onboarding others.

Reply:
I tried renaming this and it caused issues in unexpected parts of the codebase for reasons it didn't seem time-efficient to diagnose and try to resolve. I don't want to have to push the release back to accommodate this.